tools/airgap_linter.py

```diff
@@ -53,6 +53,7 @@ def is_bad_uri(uri, file_name):
         ".mesos:",
         ".mesos/",
         "$MESOS_CONTAINER_IP",
+        "$LIBPROCESS_IP",
```

Comment:
I know that this is not ideal, but we should consider the other forms of environment variable usage too:
* `${LIBPROCESS_IP}`
* `{{LIBPROCESS_IP}}` (for mustache templating)

We will also need to make this change in the `dcos-commons` repo to ensure that this is not overwritten when we update the tooling.